tests/integration/test_deploy_and_evaluate_model.py

```diff
@@ -0,0 +1,22 @@
+import integ_test_base
+import subprocess
+from pathlib import Path
+
+
+class TestDeployModelSSLOffAuthOff(integ_test_base.IntegTestBase):
```

Comment:
This class name is already used in another test file.

Reply:
thanks for catching that i forgot to update it